### PR TITLE
mobile layout for header

### PR DESCRIPTION
**Pull request by jx3yang**
## Description

Make dropdown for geyser and vault wrap to a second line when the screen gets smaller

### CODE REVIEW

File: frontend/tailwind.config.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
 // TODO: Use theme config over defining custom styles as much as possible
+const defaultTheme = require('tailwindcss/defaultTheme')
```
Comment (surudhb) [inline frontend/tailwind.config.js]: I moved boxShadow under extend so we don't lose the default classes -- so we can avoid importing the defaultTheme

File: frontend/src/components/Dropdown.tsx [hunk 2]
```diff
@@ -46,11 +46,16 @@ export const Dropdown: React.FC<Props> = ({ options, selectedOption, onChange })
...
     </Listbox>
   )
 }
 
 const Img = styled.img`
   ${tw`w-5 h-5 text-gray`}
 `
+
+const OptionsWrapper = styled.div`
+  width: 336px;
```
Comment (surudhb) [inline frontend/src/components/Dropdown.tsx]: Let's be more consistent and define a new width breakpoint and use that `w-336px`

**Review by surudhb — CHANGES_REQUESTED** [review]
NOICE! Requesting minor changes!

File: frontend/tailwind.config.js (hunk 1 above)
Comment (jx3yang) [reply frontend/tailwind.config.js]: Ok, removed the default theme, I guess you'll move the boxShadow in another commit?

File: frontend/src/components/Dropdown.tsx (hunk 2 above)
Comment (jx3yang) [reply frontend/src/components/Dropdown.tsx]: fixed

File: frontend/tailwind.config.js (hunk 1 above)
Comment (surudhb) [reply frontend/tailwind.config.js]: yessir!

**Pull request merged**